### PR TITLE
[mysql] expose meta_ts field in the metadata through flink sql

### PR DESCRIPTION
**Pull request by EMsnap**
[mysql] expose meta_ts field in the metadata through flink sql #1141

### CODE REVIEW

**Comment by EMsnap** [comment]
@leonardBang  PLAL

**Comment by EMsnap** [comment]
@PatrickRen PLAL

File: flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/table/MySqlReadableMetadata.java [hunk 1]
```diff
@@ -80,6 +80,21 @@ public Object read(SourceRecord record) {
                     return TimestampData.fromEpochMillis(
                             (Long) sourceStruct.get(AbstractSourceInfo.TIMESTAMP_KEY));
                 }
+            }),
+
+    /** it indicates the time that the data consumed by the FlinkCDC. */
+    TS(
+            "meta_ts",
```
Comment (ruanhang1993) [inline flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/table/MySqlReadableMetadata.java:87]: Maybe we need a more meaningful name like `gen_ts` or else.

Comment (ruanhang1993) [inline flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/table/MySqlReadableMetadata.java:87]: We need some tests about this new metadata.

**Review by ruanhang1993 — COMMENTED** [review]
I wonder in which situation we will use this new metadata.

**Comment by yuxiqian** [comment]
Hi @EMsnap, sorry for the delay of this PR. Could you please rebase it to latest `master` branch since there’s been lots of changes in Flink CDC repo since your original commit, and address @ruanhang1993's comments? Kindly reminder that `com.ververica.cdc.connectors.mysql` package has been moved to `org.apache.flink.cdc.connectors.mysql`.

Thanks for your contribution!

**Pull request closed without merging**